Andrea / exercise-1/index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>pagina prueba</title>
+    <link rel="stylesheet" href="stylesheet.css"
+</head>
+<body>
+    <header>
+       <nav class="top-menu">
+        <img class="logo" src="logo.png" alt="imagen">
+           <ul class="top-menu__links">
+               <li>
+                   <a href="#">features</a>
+               </li>
+               <li>
+                   <a href="#">templates</a>
+               </li>
+               <li>
+                   <a href="#">integrations</a>
+               </li>
+               <li>
+                   <a href="#">pricing</a>
+               </li>
+               <!-- Botones -->
+               <li>
+                   <a class="top-menu__button" href="#">sign in</a>
+               </li>
+               <li>
+                   <a class="top-menu__button" href="#">sign up</a>
+               </li>
+           </ul>
+       </nav>
+
+        <!-- Greetings -->
+        <section class="hero">
+            <h1>The magic of reading</h1>
+        </section>
+    </header>
+
+
+    <!-- Achievements -->
+    <section>
+        <ul class="section-menu">
+            <li>
+                <p><img src="icon.png" alt="icon">genres</p>
+            </li>
+            <li>
+                <p><img src="icon.png" alt="icon">buy</p>
+            </li>
+            <li>
+                <p><img src="icon.png" alt="icon">fanart</p>
+            </li>
+            <li>
+                <p><img src="icon.png" alt="icon">communities</p>
+            </li>
+    </section>
+
+    <main>
+        <div class="title">
+            <p><span class="title-span">read<br>more</span> open your eyes</p>
+        </div>
+        
+        <div class="body">
```

Comment:
I dont recommend using class names that could be confused accidentally with semantic elements. If I want to style this on the css I would need to do `.body { ...` but if I want to set a style on the body element it would be `body {...`. The difference is so small that anyone can make that mistake easily